### PR TITLE
Update ChangeLog

### PR DESCRIPTION
**Pull request by dcorson-ticino-com**
feature-req925 recording indicator (#968)

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
Thanks!

File: ChangeLog [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 - Escape chat messages at the server. This will no longer allow html formated chat messages due to security reasons (#939)
 
+- Highlite the fact that the server is recording in the Mixerboard title (#968), coded by dcorson-ticino.com
```
Comment (trebmuh) [inline ChangeLog:29]: Shouldn't that be "Highlight" ?

**Review by ann0see — CHANGES_REQUESTED** [review]
Could you please apply this typo fix?

**Review by ann0see — APPROVED** [review]
Thanks!

**Review by pljones — APPROVED** [review]

**Pull request merged**